tremor-cli/src/test/tag.rs

```diff
@@ -135,16 +134,12 @@ impl TagFilter {
 }
 
 pub(crate) fn resolve(base: &Path, other: &Path) -> Result<TagFilter> {
-    if let Ok(rel) = util::relative_path(base, other) {
-        let mut base = base.to_string_lossy().to_string();
-        let tags_file = format!("{}/tags.json", &base);
+    if other.strip_prefix(base).is_ok() {
+        let base_tags_file = base.join("tags.json");
         let mut tags = TagFilter::new(vec![], vec![]);
-        tags = tags.join(Some(maybe_slurp_tags(&tags_file)));
-        for dirname in rel.split('/') {
-            base = format!("{}/{}", base, dirname);
-            let tags_file = format!("{}/tags.json", &base);
-            tags = tags.join(Some(maybe_slurp_tags(&tags_file)));
-        }
+        tags = tags.join(Some(maybe_slurp_tags(&base_tags_file)));
+        let other_tags_file = other.join("tags.json");
+        tags = tags.join(Some(maybe_slurp_tags(&other_tags_file)));
```

Comment:
I think that changes the behaviour. The intention is to find all tag files in the directories between `base` and `other` this code no longer does that from what I can tell.

basically if base is `/a/b/c` and other is `/a/b/c/d/e/f` we want to look for:

- `/a/b/c/tags.json`
- `/a/b/c/d/tags.json`
- `/a/b/c/d/e/tags.json`
- `/a/b/c/d/e/f/tags.json`

Reply:
I've done some changes but I still don't think that's the right way to do it. We should probably use the globwalk crate to do this. wdyt?

Reply:
Globwalk sounds like a really good idea 👍